lib/pages/getServerSideProps/redirects.js

```diff
@@ -1,22 +1,54 @@
 const getNetlifyFunctionName = require("../../helpers/getNetlifyFunctionName");
 const getDataRouteForRoute = require("../../helpers/getDataRouteForRoute");
 const pages = require("./pages");
+const nextConfig = require("../../helpers/getNextConfig")();
 
 const redirects = [];
 
+/** getServerSideProps pages
+ *
+ * Params:
+ * route examples -> '/ssr', '/ssr/[id]'
+ * filePath examples -> 'pages/ssr.js', 'pages/ssr/[id].js'
+ *
+ * With i18n enabled:
+ * route and filePath formats are the same as above
+ *
+ **/
+
 pages.forEach(({ route, filePath }) => {
   const functionName = getNetlifyFunctionName(filePath);
+  const target = `/.netlify/functions/${functionName}`;
+
+  // If i18n, need to add extra redirects for each locale
+  // i.e. /en/ssr -> /${target}
+  if (nextConfig.i18n) {
+    const { locales = [] } = nextConfig.i18n;
+    locales.forEach((locale) => {
```

Comment:
What do you think about creating a helper `getLocales()` (or similar name), that returns the locales (as an array) or an empty array (if no i18n). So we can remove one level of nesting and squash three lines into one:

```js
getLocales().forEach(locale => {
...
})
```

Thoughts? :)